docs/gettsim_objects/input_variables.md

```diff
@@ -247,11 +247,35 @@ Weekly working hours of individual
 
 Type: float
 
+(elterngeld_claimed)=
+
+## `elterngeld_claimed`
+
+Individual claims Elterngeld
+
+Type: bool
+
+(elterngeld_nettoeinkommen_vorjahr_m)=
+
+## `elterngeld_nettoeinkommen_vorjahr_m`
+
+Approximation of the net wage in the 12 months before birth of youngest child (according
+to simplified calculation rules). You may let GETTSIM compute this variable via the
+`elterngeld_nettolohn_approximation_m` target in a separate run.
+
```

Comment:
What is the reason that this is not calculated by default by GETTSIM?

Reply:
In principle we have two options when we want to compute targets based on inputs from t-1: i) We compute everything in one GETTSIM call, i.e. the relevant target based on multiple inputs from the past period (wages, self-employed income, transfers, ... from past year / past 12 months), or ii) the user does this herself and calls GETTSIM two times. 

Hans-Martin and I agreed that the latter is much clearer than the former. It increases readability (1 input instead of 10ish, no Lohnsteuer function/module that computes Lohnsteuer just for the "Elterngeld wage" of the past 12 months, ...) and it is closer to reality (e.g. you need the income tax parameters of last year to compute the Elterngeld net wage).

Reply:
Yes, but useful to make that explicit. Also add a note somewhere (docstring of `elterngeld_nettolohn_approximation_m`, that for most datasets (i.e., all except for IAB/DRV data), the best approximation will be to just use the previous calendar year earnings, even if the earnings of the 12 months up to birth count for most people (i.e., those who only have income from dependent employment).

Reply:
It is not necessary. But it will typically be a reasonable approximation, unless you have spell data from the IAB.

Typical surveys will give you annual income in the past calendar year. For a kid born in 2019, you could use 2018 incomes or 2019 incomes. I would think that in the vast majority of cases, the error using 2018 incomes (and parameters) will be smallest because for a kid born in January 2019, you likely won't see any 2019 labor income etc.. So you'd go ahead and calculate `elterngeld_nettolohn_approximation_m` with 2018 data and policy env, then `elterngeld_m` in 2019.

But if you want to be super-precise, you could of course run `elterngeld_nettolohn_approximation_m` using the 2019-01-01 policy environment and 2018 data; then run it another time to calculate `elterngeld_m`.

Reply:
Sorry, I still do not understand the core of the argument.

Why is requiring `elterngeld_nettolohn_approximation_m` better than requiring `elterngeld_bruttolohn_reference_period_m` as an input?

Reply:
From my understanding:

1. You wouldn't need only the Bruttolohn but also an array of other variables (i.e. lagged values of all variables necessary to compute Lohnsteuer).
2. Depending on which parameters are necessary, you'd also need historical parameters in the GETTSIM call.

Reply:
> You wouldn't need only the Bruttolohn but also an array of other variables (i.e. lagged values of all variables necessary to compute Lohnsteuer).

Which input variables are you thinking about? For all input variables I can come up with, only the state of the household at the time of birth matters. Withholding tax class is complicated as it is unclear which one is applied to the indvidiual if the household picked a different schedule between interviews.

> Depending on which parameters are necessary, you'd also need historical parameters in the GETTSIM call.

Which ones?

Reply:
I think it would make for an interesting case study (maybe an MSc thesis in conjunction with your co-author, Jakob?) to find out which approximation would work best with a reasonable amount of work. After all, not everybody is paying Lohnsteuer only (at least those PV-things don't cause income tax to be used anymore...) and we would need to think of them, too. So we will never be able to be extremely precise in all use cases and I would guess most of the stuff we are discussing here is about eliminating second-order errors when we might still make (data-driven) first-order errors. But hard to find out by just discussing and without anyone actually crunching the data -- I'd certainly be happy to be proven wrong on this one!

Reply:
> For all input variables I can come up with, only the state of the household at the time of birth matters.

I didn't know that! Is that also true for place of residence (east/west) and Kinderfreibeträge (for Soli)?

Reply:
The application form states: "Die entsprechenden Abzugsmerkmale werden den Lohn- oder Gehaltsabrechnungen entnommen. Grundsätzlich sind die Abzugsmerkmale maßgeblich, die in der überwiegenden Zahl der Monate des Bemessungszeitraums vor Geburt des Kindes gegolten haben."

Reply:
So we would need an array of historical parameters, right?

Reply:
Only if a change in the parameters happened less than 6 months ago. But I'm convinced that it is good as it is.

Reply:
Cool. Let's trust the law and 3 concurrent interpretations of that (including one real expert in @pia-molitor!) and consider this discussion closed for good. Will resolve so that it does not confuse people.